core/users.js

```diff
@@ -0,0 +1,51 @@
+import configs from 'configs/index';
+import jwt from 'jsonwebtoken';
+import encryption from 'utils/encryption';
+import { UserNotFoundError, PasswordNotMatchError } from 'common/error';
+import User from 'db/models/User';
+
+const secretKey = configs.SECRET_KEY;
```

Comment:
No need to use a variable. Using `configs.SECRET_KEY` where you need it is enough

Reply:
Fixed

Reply:
https://github.com/EasyEventV2/ee2_api/pull/1/commits/73716262368593e9f1d6f9ac94ab472189fd237b